### PR TITLE
Remove networking-istio and gateways from 0.12.1.yaml

### PR DESCRIPTION
**Pull request by nak3**
This patch changes to:

- remove networking-istio deployment
- remove cluster-local-gateway and knative-ingress-gateway

from 0.12.1 manifests.

This change is testing with https://github.com/openshift-knative/serverless-operator/pull/116.

### CODE REVIEW

**Comment by nak3** [comment]
https://github.com/openshift-knative/serverless-operator/pull/113 is testing this change.

**Comment by nak3** [comment]
travis does not set up with Kourier, so it is failing...

**Comment by nak3** [comment]
Actually, `test/e2e-test.sh` is broken in release-0.12 branch. https://travis-ci.com/openshift-knative/serving-operator/builds/147756098 also failed with `test/e2e-travis.sh: line 23: knative_setup: command not found`.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

/hold

Can we merge this once all of the ServiceMesh migration logic landed? This isn't harmful in the cluster while we shift stuff over to Kourier, right?

File: cmd/manager/kodata/knative-serving/0.12.1.yaml
Comment (markusthoemmes) [inline cmd/manager/kodata/knative-serving/0.12.1.yaml:1872]: Can we remove everything that has this label from the manifest?

Comment (nak3) [reply cmd/manager/kodata/knative-serving/0.12.1.yaml:1872]: Sure, I have removed ClusterRole and Configmap for networking-istio, that have the label.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve
/hold cancel

**Pull request merged**
